drivers/SmartThings/zigbee-window-treatment/capabilities/initializedStateWithGuide.yaml

```diff
@@ -0,0 +1,20 @@
+id: stse.initializedStateWithGuide
```

Comment:
can't you just use your _other_ initialized state custom capability

we'd really prefer to add as few custom capabilities into _our_ repo as possible

Maybe, if this is installation instructions, it would be better to include in the catalog info on installation?

Reply:
Hi @greens , it is related with plugin feature. let me give you more details on slack

Reply:
Ideally, for tiny revisions to existing capabilities you have already published, the best course of action might just be to update those with any changes you need rather than creating another new, largely identical, capability. This is, as you can see, leading to a drastic increase in your code complexity.